website/docs/components/table/advanced-table/partials/accessibility/accessibility.md

```diff
@@ -12,7 +12,86 @@ The table row element cannot receive interactions, meaning actions cannot be att
+
+## Keyboard Navigation
+
+The Advanced Table supports two keyboard interaction modes: Navigation Mode and Action Mode. These modes allow users to move through table data and interact with elements using a keyboard.
+
+### Navigation Mode
+
+
+<Doc::Badge @type="neutral">Tab</Doc::Badge>
+
+When a user tabs into the Advanced Table, the first cell is focused.
+
+![Keyboard focus on the sortable 'Artist' column header with sorting controls active.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-tab.png)
+
+
+<Doc::Badge @type="neutral"><Hds::Icon @name="arrow-left" @title="Left arrow" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-right" @title="Right arrow" /></Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-up" @title="Up arrow" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-down" @title="Down arrow" /></Doc::Badge>
+
+Arrow keys move focus to cells in the direction of the input. If the table extends beyond the screen, arrow keys allow movement to access hidden columns.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-arrow-key-down.png)
+
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-left" @title="Left arrow" /></Doc::Badge>
+
+Move focus to the first cell in the row.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-left.png)
+
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-right" @title="Right arrow" /></Doc::Badge>
+
+Move focus to the last cell in the row.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-right.png)
+
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-up"  @title="Up arrow" /></Doc::Badge>
+
+Move focus to the first cell in the column.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-up.png)
+
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-down"  @title="Down arrow" /></Doc::Badge>
+
+Move focus to the last cell in the column.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-down.png)
+
+
+### Action Mode
+
+<Doc::Badge @type="neutral">Enter</Doc::Badge>
+
+Press Enter in Navigation Mode to enable Action Mode and interact with elements inside a cell. Focus will move to the first actionable element.
```

Comment:
You say "inside a cell", but action mode works in any table element (and you use an image of a header in the next example). Should this be more generic?

Reply:
I think Andrew is correct with the current wording - with the advanced table you cant focus anything inside a cell until you press enter. The Advanced Table is by default 1 tab stop (so users can skip past it if they want). When you press enter you focus the first interactive element inside the cell and your focus is trapped within that cell until you press escape.

I agree it would be more consistent to use an image of a header cell here to follow with the rest of the action mode images, but definitely not a blocker to me.